test/integration/ethereum/addresses.ts

```diff
@@ -45,7 +45,7 @@ export const PRODUCTION_ADDRESSES = {
       router: "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D",
     },
     uniV3: {
-      router: "0xE592427A0AEce92De3Edee1F18E0157C05861564",
+      router: "0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45",
```

Comment:
We should double check that this address is not used elsewhere, where we still need to use the old router version.